Documentation/security/network/encryption-ipsec.rst

```diff
@@ -306,6 +306,8 @@ To disable the encryption, regenerate the YAML with the option
 Limitations
 ===========
 
+    * For clusters running in native routing mode, IPsec encryption is not applied to
+      connections which are selected by a L7 Egress Network Policy or a DNS Policy.
```

Comment:
nit: I saw both `a L7` and `an L7` are using in cilium, not sure which one is correct :rofl: